docs/source/dev_install.md

```diff
@@ -0,0 +1,73 @@
+Developer Install
+=================
+
+<script>
+w({
+  "e43a868c5c0b4db39f0416d0fe9cc476": {
+    "model_name": "IntSliderModel",
+    "model_module": null,
```

Comment:
If you rebase this on master, you will probably need to replace the model-module with jupyter-js-widgets.